### ISSUE TITLE
Add non-commercial licenses

### ISSUE THREAD

**Issue by EwoutH** [github/choosealicense.com#1215]
Currently all licenses allow commercial use. While in many cases that’s fine, there is a use case in non-commercial licenses, and it would be nice to have a few options listed.

One use case would be making source code available for individuals and researchers, but don’t allow commercial use by default. In some cases this might be combined with more permissive licensing on case/contract basis. 

Having a non-commercial license is in many cases more free than “No license” situation.

Two potential options to add are:
- [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0)
- [The Commons Clause](https://commonsclause.com/)

**Comment by javf88** [comment]
I do support the motion inhere.
Is there any legal restriction if I just add it?
The fact that the GitHub does not list the BY-NC 4.0 does not mean I cannot add such a license right?

It would be nice also to clarify this.

**Comment by tyoc213** [comment]
As far as I know you can put whatever text you want in `LICENSE`

**Comment by stefanocoretta** [comment]
Non-commercial licenses are for example very useful for textbooks! I got a textbook on GitHub and GitHub pages. The textbook is free and will always be free. I do not wish for someone to sell it in print and profit from it.

Of course I added a CC-BY-SA-NC license but the license API picks it up as other because it’s not in the official list.

**Comment by Tarekchehahde** [comment]
Non-commercial licenses such as **CC BY-NC 4.0** and the **Commons Clause** do not fit choosealicense.com’s mission or [contribution criteria](https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-a-license):

- CC BY-NC is not an [Open Definition](https://opendefinition.org/) conformant license (NC restrictions).
- The Commons Clause is not an SPDX-listed license and is not on the OSI/FSF/OD approval lists the project requires.
- The site focuses on licenses suitable for **open source software** repositories, not NC / source-available terms.

No PR planned for this issue; `/non-software/` already points readers to Creative Commons for some non-code use cases under approved licenses.
